### DISCUSSION TITLE
Bot doesn't detect mobs

### DISCUSSION THREAD

**Discussion by Scobrant** [PrismarineJS/mineflayer#3082] [Q&A]
So I made a bot with minelayer and added guard example in it but it doesn't detect mobs like zombie etc. except phantom
It only hits phantom while guarding the location.

Server 1.19.3 Paper
node: 18.16.0
code: 
https://github.com/UzayliGAMER/my-code.git
(don't mind Turkish things in the code)

**Comment by SonicandTailsCD** [comment]
I'm sorry but I cannot see your code. [My browser said it's not available.] Do you mind sharing it here? :/
